src/main/scala/io/epiphanous/flinkrunner/model/sink/SinkConfig.scala

```diff
@@ -34,7 +34,7 @@ import scala.reflect.runtime.{universe => ru}
   */
 trait SinkConfig[ADT <: FlinkEvent] extends SourceOrSinkConfig[ADT] {
 
-  override val _sourceOrSink = "sink"
+  override def _sourceOrSink = "sink"
```

Comment:
Why did you change this to a def?

Reply:
why is that a part of your PR @sbhatt-mdsol ?

Reply:
May be because I have based my branch out of Bob's  `rfl/fix-kinesis-source` branch not from `main` which has been merged now and I doesn't exist any more.  I will open a new PR to see it reflects properly.